src/components/Person/Person.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import './Person.scss';
+
+const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = { ...person };
+  const partnerSex = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age ? <p className="Person__age">{`I am ${age}`}</p> : null}
+      {!isMarried
+        ? <p className="Person__partner">I am not married</p>
+        : (
+          <p className="Person__partner">
+            {`My ${partnerSex}'s name is ${partnerName}`}
+          </p>
+        )
```

Comment:
Don't repeat yourself: <p className="Person__partner"> repeated in your code
It's better to create one more variable to store string, that depends on isMarried property and use it in this paragraph